API Blueprint Specification.md

```diff
@@ -583,7 +584,7 @@ Resource representing **ACME Blog** posts.
 <a name="def-model-section"></a>
 ## Resource model section
 - **Parent sections:** [Resource section](#def-resource-section)
-- **Nested sections:** [Refer to payload section](#def-payload-section)
+- **Nested sections:** [Refer to payload section](#def-payload-section), [`0-1` **URI Parameters** section](#def-uriparameters-section)
```

Comment:
Do we really want to add URI Parameters section to a resource model? Since a resource model doesn't have an associated URI (until it's referenced).